docs/git/index.md

```diff
@@ -0,0 +1,220 @@
+
+The file you opened, `README.md`, has a question in it. We're going to answer the question, and submit it as if it was our homework.
+
+We can edit the file in VS Code to answer the question, and save it like normal.
+
+After we've saved the file, if we open up Github Desktop, something interesting has changed. On the left, it now says "1 changed file", and on the right, it shows us what the change was. Things we've removed get a red background, and things we've added get a green background.
+
+This is a really useful way for us to check over our homework before submitting it. If we've accidentally deleted things, or changed things we didn't mean to, we can notice now, and un-do them by editing the file again.
+
+When we're happy with our change, we can press the "Commit to main" button. That tells Git "This change is a good change, I want to keep it".
+
+Now, because we committed a change, our copy of the repository is different from CodeYourFuture's copy. Github Desktop now gives us a new button: "Push origin" ("origin" is what Git calls "where I cloned this repository from"). If we press it, it will try to send our change to the CodeYourFuture version.
+
+We didn't have this button before, because we hadn't committed any changes. There's an important distinction here: When we save files in VS Code, we store them in the file on our computer, but Git doesn't automatically commit them. It notices the changes (it showed them to us!), and asks us if we want to commit them. TODO: I'm not in love with any ways of describing "when to commit"...
+
+So, we're happy with our homework, we've committed it, let's try pushing it!
+
+## Forking
+
+Github Desktop just gave us a really useful warning, but it uses some words we haven't seen yet!
+
+We just tried to push our changes to CodeYourFuture's copy of this repository. But we're not allowed to do that! Imagine if anyone could just push anything they wanted to our repository! Then someone naughty could delete the homework, or worse.
+
+To avoid this, we use something called a _fork_ and something called a _pull request_.
+
+Remember when we cloned the repo, we pasted (TODO: URL) in as the place to clone from? Let's look at that:
+
+TODO: Insert image annotating bits of the URL
+
+This is saying "On Github, the user CodeYourFuture has a repository called TODO, I want that".
+
+Github lets you host your own copy of the repository on Github too! If your username is EagerLearner, can you guess what URL your repository would be at?
+
+That's right, https://github.com/EagerLearner/TODO!
+
+This is called a _fork_. It's a copy of the repository, where you're allowed to make changes. So when Github Desktop just asked us "Do you want to fork this repository?", what it's really saying is "You're not allowed to make changes to CodeYourFuture's repository, would you like to make your own copy on Github where you _are_ allowed to make changes, and put your changes there?"
```

Comment:
From @40thieves:

I like this "plain English" approach, but I wonder if it's a bit buried in this long paragraph?

I'm imagining an image of some speech bubbles, showing the equivalence. Does that make sense?

Reply:
Totally - additionally, I'm hoping the graphic described in the TODO below showing the 4 different copies will help make this clear too